presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/aggregation/TestGeometryStateSerializer.java

```diff
@@ -28,11 +29,19 @@
 
 public class TestGeometryStateSerializer
 {
+    private AccumulatorStateFactory<GeometryState> factory;
+    private AccumulatorStateSerializer<GeometryState> serializer;
+
+    @BeforeMethod
```

Comment:
We can use `@BeforeClass`. Don't forget to nullify them in the `@AfterClass(alwaysRun=true)`

Reply:
Then the same factory and serializer would be used by every test. Wouldn't that cause some issues?

Reply:
We parallelize our tests on the method level. From what I know, TestNG reuses the same class instance when run methods in parallel. `@BeforeMethod` hooks might interfere with each other in such case.